sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -303,19 +303,19 @@ export class ServiceBusManagementClient extends ServiceClient {
     deleteTopic(topicName: string, operationOptions?: OperationOptions): Promise<Response>;
     getNamespaceProperties(operationOptions?: OperationOptions): Promise<NamespacePropertiesResponse>;
     getQueue(queueName: string, operationOptions?: OperationOptions): Promise<QueueResponse>;
-    getQueueRuntimeInfo(queueName: string, operationOptions?: OperationOptions): Promise<QueueRuntimeInfoResponse>;
+    getQueueRuntimeProperties(queueName: string, operationOptions?: OperationOptions): Promise<QueueRuntimePropertiesResponse>;
     getQueues(options?: ListRequestOptions & OperationOptions): Promise<QueuesResponse>;
-    getQueuesRuntimeInfo(options?: ListRequestOptions & OperationOptions): Promise<QueuesRuntimeInfoResponse>;
+    getQueuesRuntimeProperties(options?: ListRequestOptions & OperationOptions): Promise<QueuesRuntimePropertiesResponse>;
```

Comment:
There is a sample in the samples/advanced folder that would also need the update.

Reply:
Ah! I made the update to the ts one, and missed the js one :) 
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/10083/commits/f6291bb6210025398aedab765b30c8e14ea58e79